setup.py

```diff
@@ -119,7 +119,7 @@
 ]
 
 extra_deps['megablocks'] = [
-    'megablocks<1.0',
+    'megablocks[gg]<1.0',
```

Comment:
should either have the gg extra _or_ the grouped_gemm dep here, not both.

Reply:
I'm not sure I follow...